reducers/plugins/index.ts

```diff
@@ -24,9 +30,9 @@ function hasMenuId(menu, menuId) {
     return false;
 }
 
-function buildMenu(rootMenu, data) {
+function buildMenu(rootMenu: any, data: any) {
```

Comment:
Could do the same here, not sure if this will work exactly but it should be something like this:
```suggestion
function buildMenu(rootMenu: Menu, data: Menu) {
```